cmake/tribits/core/package_arch/TribitsDefineStandardCompileVars.cmake

```diff
@@ -108,14 +108,8 @@ MACRO(TRIBITS_DEFINE_STANDARD_COMPILE_FLAGS_VARS  ENABLE_SHADOWING_WARNINGS)
   IF (${PROJECT_NAME}_CXX_STRONG_COMPILE_WARNING_FLAGS)
     SET(CXX_STRONG_COMPILE_WARNING_FLAGS ${${PROJECT_NAME}_CXX_STRONG_COMPILE_WARNING_FLAGS})
   ELSE()
-    IF (NOT ${PROJECT_NAME}_ENABLE_CXX11)
-       SET(CXX_98_OPTION " -std=c++98") # C++98 standard code
```

Comment:
Right, this was to enforce the C++98 standard for code we wanted to make sure did not allow to skip in C+11 features.  No need for that anymore.  In fact, this should have been removed a long time ago when Trilinos was made to require C++11.